runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,68 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+get({opts})                                         *vim.lsp.inlay_hint.get()*
+    Get the list of inlay hints, (optionally) restricted by buffer, client, or
+    range. This method does not make an LSP request, but just queries locally
+    stored inlay hints. The returned inlay hints are unresolved.
+
+    Example usage: >lua
+        local timeout, bufnr = 100, vim.api.nvim_get_current_buf()
+        local hint = vim.lsp.inlay_hint.get({ bufnr = bufnr })[1]
+
+        local client = vim.lsp.get_clients({ id = hint.client_id })[1]
```

Comment:
```suggestion
        local client = vim.lsp.get_client_by_id(hint.client_id)
```

Reply:
Thanks for the tip!

Reply:
This makes me realize that `get_client_by_id` might be redundant? Given the flexibility of the filter that can be used with `get_clients`, why keep the specialized functions?